build-system/tasks/extension-helpers.js

```diff
@@ -464,18 +466,12 @@ async function buildExtension(
   }
   const extDir = 'extensions/' + name + '/' + version;
 
-  // Use a separate watcher for extensions to copy / inline CSS and compile JS
-  // instead of relying on the watchers used by compileUnminifiedJs and
-  // compileMinifiedJs, which only recompile JS.
+  // Use a separate watcher for css and jison compilation.
+  // The watcher within compileJs recompiles the JS.
```

Comment:
FMI: How does css / jison recompilation work? Does this watcher trigger a rebuild of css / jison to js, after which the watcher within `compileJs` detects the change in that js output file?

Reply:
Correct. This watcher only triggers compilation of the css/jison files. The watcher within `compileJs` is in charge of recompiling whatever deps changed